wms/i3.js

```diff
@@ -0,0 +1,77 @@
+import { createClient } from 'i3';
```

Comment:
This proved a bit hard to do. I think we need to rethink/restructure the flow types a bit.